editor.planx.uk/src/@planx/components/FindProperty/Public/index.tsx

```diff
@@ -204,6 +211,10 @@ function Component(props: Props) {
           if (regions) {
             newPassportData["property.region"] = regions;
           }
+          if (titleBoundary) {
+            newPassportData["property.boundary.title"] = titleBoundary;
+            newPassportData["property.boundary.title.area"] = Number(area(titleBoundary as any).toFixed(4));
```

Comment:
todo: align rounding & units to `formatArea` helper method in map repo https://github.com/theopensystemslab/map/blob/45fa7284c7b0f3d15b2aadb69d64807d42749c77/src/components/my-map/utils.ts#L21

passport should store area of title boundary as both square metres & hectares similar to site boundary?